region/maridia/inner-green/West Sand Hall.json

```
@@ -376,9 +376,7 @@
         "canCarefulJump"
       ],
       "note": [
-        "Gain momentum in the room to the left, then jump in this room in order to land on the first pillar. Ensure that Speed Booster is disabled.",
-        "With a short runway of 5 tiles in the adjacent room, it is required to jump as late as possible at the edge of the runway in this room.",
-        "With any longer of a runway, the jump is much less precise."
+        "Gain momentum in the room to the left, then jump in this room in order to land on the first pillar. Ensure that Speed Booster is disabled."
```

```suggestion
        "Gain momentum in the room to the left, then jump after entering this room in order to land on the first pillar. Ensure that Speed Booster is disabled."
```
I feel like this point should be extra explicit, since it's not obvious and could be confusing since the strat uses `canCrossRoomJumpIntoWater`